apple-pie/index.html

```diff
@@ -3,19 +3,54 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./styles/style.css">
     <title>Apple Pie Recipe</title>
     <!-- don't forget to link your styles -->
   </head>
   <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! Ingredients 1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter 3 tablespoons
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
-    C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. Place the bottom crust in your pan. Fill with apples, mounded
-    slightly. Cover with a lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-    that it does not run off. Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-    degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
+    <header>
+      <h1>Apple Pies</h1>
+      
+    </header>
+    
```

Comment:
Agregue iun contenedor para tus div de parrafos y contenido despues de la imagen y el titulo tambien saque del div de ingredientes tu imagen 
```suggestion
    <div class="container">
      <div class="info">
        <p>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
          be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
          favorites as well! </p>
          <img src="./images/recipe-info.png" alt="">
        </div>
      </hr>
    <div class="ingredientes">
      <h2> Ingredients</h2>
      <ul>
        <li>1recipe pastry for a 9 inch double crust pie</li>
        <li>/2 cup unsalted butter</li>
        <li>3 tablespoons
          all-purpose flour</li>
        <li>1/4 cup water</li>
        <li>3 tablespoons all-purpose flour</li>
        <li>1/4 cup water</li>
        <li> 1/2 cup white sugar</li>
        <li>1/2 cup packed
          brown sugar</li>
        <li> 8 Granny Smith apples - peeled, cored and sliced </li>
      </ul>
    </div>
    <img src="./images/cooking-info.png" alt="">
    <hr/>
    <div class="directions">
    <h2>Directions</h2>
    <ol>
      <li>Preheat oven to 425 degrees F (220 degrees
        C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
        to a boil. Reduce temperature and let simmer.</li>
      <li>Place the bottom crust in your pan. Fill with apples, mounded
        slightly. Cover with a lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
        that it does not run off.</li>
      <li>Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
        degrees C). Continue baking for 35 to 45 minutes, until apples are soft.</li>
    </ol>
  </div>
    </div>
    
              
  </body>
</html>

```